src/openforms/emails/migrations/0010_add_cosign_templatetag.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.2.19 on 2023-06-15 09:55
+
+from django.db import migrations
+from django.db.models import Value, Func, TextField, F
+
+from openforms.utils.regex import add_cosign_info_templatetag
+
+
+def add_cosign_template_tag_to_email_confirmation_template(apps, _):
+    ConfirmationEmailTemplate = apps.get_model("emails", "ConfirmationEmailTemplate")
+
+    templates = ConfirmationEmailTemplate.objects.all()
+    if not templates.exists():
+        return
+
+    for template in templates:
+        template.content = add_cosign_info_templatetag(template.content)
```

Comment:
At some point we could do this with one query: https://github.com/django/django/pull/12438